invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordManagement.js

```diff
@@ -27,28 +26,29 @@ export const RecordManagement = (props) => {
   return (
     <Grid relaxed>
       <Grid.Column>
-        <Grid.Row>
-          <Icon name="cogs" />
-          <h2 id="record-management-header">{i18next.t('Manage')}</h2>
-        </Grid.Row>
         <Grid.Row className="record-management-row">
           <EditButton recid={recid} onError={handleError} />
+        </Grid.Row>
+        <Grid.Row className="record-management-row">
           <NewVersionButton
+            style={{ display: "block" }}
+            fluid
             record={record}
             onError={handleError}
             disabled={!permissions.can_new_version}
           />
+        </Grid.Row>
+        <Grid.Row className="record-management-row">
           {permissions.can_manage && (
             <ShareButton
-              divClassName="share-button"
               disabled={!permissions.can_update_draft}
               recid={recid}
             />
           )}
         </Grid.Row>
         {error && (
           <Grid.Row className="record-management-row">
-            <p>{error}</p>
+            <Message negative>{error}</Message>
```

Comment:
please add to your issue to replace this also with a generic error component, we keep using `<message negative` in several different ways, so it would be nice to harmonize it